### ISSUE TITLE
rollup --config 'module-from-npm'

### ISSUE THREAD

**Issue by bfred-it** [rollup/rollup#736]
For the sake of sharable/reusable configurations, it'd be nice to be able to require a config file from npm:

```sh
npm install rollup-config-buble-uglify --save-dev
rollup --require-config 'buble-uglify' input.js > output.js
```

Obviously this comes at the expense of configurability, but it'd be nice to have some general-use boilerplate.

It's already kinda possible, so this issue is more about discussing `rollup-config-*` type of modules.

```sh
# current solution
rollup --config 'node_modules/rollup-config-buble-uglify/index.js' input.js > output.js
```

**Comment by Rich-Harris** [comment]
Maybe a convention like `rollup -c node:buble-uglify`?

**Comment by Rich-Harris** [comment]
Have opened a PR for this https://github.com/rollup/rollup/pull/739

**Comment by bfred-it** [comment]
Perfect! I'm going to test it today, I had already published a simple `rollup-config-buble` to npm

**Comment by Rich-Harris** [comment]
available in 0.32.1

**Comment by bfred-it** [comment]
:heart:
